templates/internet-of-things/index.html

```diff
@@ -469,7 +469,7 @@ <h3 class="p-card__title">Qualcomm Dragonboard</h3>
     <div class="col-6 p-card">
       <header class="no-border u-align--center u-vertically-center u-hide--medium u-hide--small" style="min-height: 7rem;">
         {{ image (
-          url="https://assets.ubuntu.com/v1/f8ef46d3-AMD_Xilinx_logo.svg",
+          url="https://assets.ubuntu.com/v1/327b8104-AMD_E_Blk_RGB.png",
```

Comment:
This logo looks a bit stretched 
![image](https://github.com/canonical/ubuntu.com/assets/17607612/af1220a2-918d-4b2f-86f7-7f38e78fccba)

Reply:
Thanks!